src/main/java/org/rrd4j/graph/ValueScaler.java

```diff
@@ -16,6 +16,9 @@ class ValueScaler {
     }
 
     Scaled scale(double value, boolean mustRescale) {
+        // avoid NaN in legend
```

Comment:
NaN is not 0, so it should be kept in the legend.

Reply:
You're right of course, but for our less-technical users in multiple languages, 0 makes more sense than NaN. You can skip it if you don't want this change.